templates/libraries/jersey3/api.mustache

```diff
@@ -16,12 +16,20 @@ import java.util.Map;
 
 {{#operations}}
 public class {{classname}} extends Service {
+
+    public static final String API_VERSION = "{{{version}}}";
```

Comment:
Takes the value of [`#/info/version`](https://swagger.io/specification/#info-object).

Reply:
Do you need this to construct your baseUrl in a maintainable fashion?

Reply:
Yes exactly, I suppose your SDK knows which version of the API it supports, so having it available here would allow us rely on it instead of hardcoding it ourselves.

Reply:
Yes, we use API specs which are tightly coupled to the actual API. 👍